modules/backend/formwidgets/repeater/assets/js/repeater.js

```diff
@@ -144,8 +144,11 @@
     }
 
     Repeater.prototype.onAddItemSuccess = function(ev) {
-        this.togglePrompt()
-        $(ev.target).closest('[data-field-name]').trigger('change.oc.formwidget')
+        var self = this;
+        setTimeout(function () {
+            self.togglePrompt();
+            $(ev.target).closest('[data-field-name]').trigger('change.oc.formwidget')
+        }, 10);
```

Comment:
Would someone be willing to give this is a test?
```suggestion
        window.requestAnimationFrame(() => {
            this.togglePrompt();
            $(ev.target).closest('[data-field-name]').trigger('change.oc.formwidget')
        });
```

Reply:
@bennothommo That works as well.